common/app/views/fragments/inlineJSNonBlocking.scala.html

```diff
@@ -23,6 +23,11 @@
     @InlineJs(showUserName().body, "showUserName.js")
 }
 
+@if(enableConsentManagementService.isSwitchedOn) {
+    // add CMP stub and attach main script to the head
+    @InlineJs(prepareCmp().body, "prepareCmp.js")
```

Comment:
@sndrs not sure if this is the best approach, is this okay for now? I don't know much about the inlineJS and head of the document! 🙈 🙊 

I am hoping we can end up starting up the CMP without adding the stub. However if we do end up using it for consent management beyond commercial, maybe we should have it here?